spec/dummy/config/environments/test.rb

```diff
@@ -13,8 +15,22 @@
   config.eager_load = false
 
   # Configure static asset server for tests with Cache-Control for performance.
-  config.serve_static_assets  = true
-  config.static_cache_control = "public, max-age=3600"
+  cache_header_value = 'public, max-age=3600'
+
+  if config.respond_to?(:public_file_server)
```

Comment:
If this is a Rails 4/5 thing, I think you should comment which is which for your future sanity and ripping out support for 4.

Reply:
oh nevermind this is spec/dummy, I don't care that much